db/migrate/20230524122135_extend_policy_chain_my_sql.rb

```diff
@@ -0,0 +1,7 @@
+class ExtendPolicyChainMySql < ActiveRecord::Migration[5.2]
+  # Oracle and PostgreSQL do not need this as their `policies_config` has unlimited size
+
+  def change
+    change_column :proxies, :policies_config, :mediumtext if System::Database.mysql?
```

Comment:
I guess you have tried this working with `show create table`. Too bad that I didn't use in the other migration and used size instead.

Reply:
yes, this is the resultim schema:

`t.text "policies_config", limit: 16777215`

and the resulting DDL:

`'policies_config' mediumtext COLLATE utf8_bin`